src/objects/furniture/BaseFurniture.tsx

```diff
@@ -56,6 +57,15 @@ export class BaseFurniture
     this._refreshFurniture = true;
   }
 
+  public get alpha() {
+    return this._alpha;
+  }
+
+  public set alpha(value) {
+    this._alpha = value;
+    this._updateFurniture();
```

Comment:
I think it would be better to mark the furniture as "to be updated" with this code:
```suggestion
    this._refreshFurniture = true;
```

Before the next frame, it will execute the `_updateFurniture` method for you.